Resources/Prototypes/Entities/Objects/Power/cable_coils.yml

```diff
@@ -10,31 +9,40 @@
   name: cable stack
   suffix: Full
   components:
-  - type: Stack
-    stacktype: enum.StackType.Cable
-  - type: Sprite
-    sprite: Objects/Tools/cables.rsi
-  - type: Item
-    sprite: Objects/Tools/cables.rsi
-  - type: WirePlacer
-  - type: Clickable
+    - type: Stack
+      stacktype: enum.StackType.Cable
+    - type: Sprite
+      sprite: Objects/Tools/cables.rsi
```

Comment:
Can probably netsync false here; if not then apply it to all children instead.

Reply:
ok, will netsync it.